README.md

```diff
@@ -31,6 +31,8 @@ also feel free to reach out to discuss.
 
 See also our [contributor guide](CONTRIBUTING.md) and the Kubernetes [community page](https://kubernetes.io/community) for more details on how to get involved.
 
+If you want to use Karpenter to your own cloud provider, see [Porting Karpenter to your cloud provider](docs/port-karpenter-on-our-cloud.md).
+
```

Comment:
Maybe a supported cloudproviders section here makes sense? Ideally one could look to see if their provider is implemented before diving into it here? 


**Supported CloudProviders**

| Provider | Provider Code | Provider Docs | 
| aws | https://github.com/aws/karpenter-provider-aws | https://karpenter.sh/ | 
| azure | https://github.com/Azure/karpenter-provider-azure | https://aka.ms/aks/nap, <todo proper karpenter.sh style site| 
| kwok | https://github.com/kubernetes-sigs/karpenter/blob/main/kwok/main.go | https://github.com/kubernetes-sigs/karpenter/blob/main/kwok/README.md |

Docs are still in negotiations as to how we should share docs and if we should have provider specific websites, but linking to the code is valuable. Any new engineer trying to implement a cloud provider will benefit from looking at the other implementations of the cloudprovider interfaces.